index.js

```diff
@@ -2,14 +2,80 @@ const Vis = require("./graphvis");
 const { Module, render } = require("./visRenderer");
 
 let Sequelize;
-const relationship = ({ source, target, associationType, as }) => {
-  const typeString = {
-    BelongsTo: `[arrowtail=crow, arrowhead=none, dir=both, arrowsize=0.60]`,
-    BelongsToMany: `[arrowtail=crow, arrowhead=crow, dir=both, arrowsize=0.60]`
-  }[associationType];
-  if (typeString) {
-    return `"${source.name}" -> "${target.name}" ${typeString}`;
-  }
+
+const relationships = (associations, arrowShapes = {}, arrowSize = 0.6) => {
+  const mappings = [];
+
+  const mapper = {
+    associationTypes: new Map([
```

Comment:
Why are we using a `Map` here?

Reply:
Could we replace this section with

```
const associationTypes = {
  ...defalultAssociationTypes,
  ...arrowShapes
}
```

?

Reply:
I feel like we probably could do that! I'll take a look when I'm able and report back.

Reply:
In the most recent commit, I simplified this to not use `Map`